homeworks/lesson2/app/src/components/login/login.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
```

Comment:
Why the component name is LoginForm but file name as login?
Be more consistent in your code.

Reply:
Updated